Cargo.toml

```
@@ -31,6 +31,8 @@ log = "0.4.21"
 toml = "0.8.15"
 lazy_static = "1.5.0"
 hex = "0.4.3"
+reqwest = { version = "0.12.5", features = ["json"] }
+thiserror = "1.0.63"
 
 [dev-dependencies]
 async-trait = "0.1.80"
```

I think these 2 dependencies are only for tests, so could be put as `dev-dependencies`?

```suggestion

[dev-dependencies]
async-trait = "0.1.80"
reqwest = { version = "0.12.5", features = ["json"] }
thiserror = "1.0.63"
```

---

problem then is:

```
   Compiling telos-translator-rs v0.1.0 (/home/g/repos/telos-new-translator-rs)
error[E0432]: unresolved import `reqwest`
 --> src/test_utils.rs:1:5
  |
1 | use reqwest::Client;
  |     ^^^^^^^ use of undeclared crate or module `reqwest`

error[E0432]: unresolved import `thiserror`
 --> src/test_utils.rs:3:5
  |
3 | use thiserror::Error;
  |     ^^^^^^^^^ use of undeclared crate or module `thiserror`

error: cannot find attribute `error` in this scope
  --> src/test_utils.rs:32:7
   |
32 |     #[error("HTTP error: {0}")]
   |       ^^^^^

error: cannot find attribute `from` in this scope
  --> src/test_utils.rs:33:17
   |
33 |     HttpError(#[from] reqwest::Error),
   |                 ^^^^

error: cannot find attribute `error` in this scope
  --> src/test_utils.rs:34:7
   |
34 |     #[error("API error: {0}")]
   |       ^^^^^

error[E0433]: failed to resolve: use of undeclared crate or module `reqwest`
  --> src/test_utils.rs:33:23
   |
33 |     HttpError(#[from] reqwest::Error),
   |                       ^^^^^^^ use of undeclared crate or module `reqwest`

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `telos-translator-rs` (lib) due to 6 previous errors
```

Seems we need to change the `test_utils` module encapsulation in order to make those dev deps, tried using `#[cfg(test)]` statement on `lib.rs` to make the module be present only on test builds but then I get this:

```
error[E0432]: unresolved import `telos_translator_rs::test_utils`
  --> tests/mock_fork.rs:13:26
   |
13 | use telos_translator_rs::test_utils::{LeapMockClient, SetJumpsParams};
   |                          ^^^^^^^^^^ could not find `test_utils` in `telos_translator_rs`
   ```